lib/Steps/SentryProjectSelector.ts

```diff
@@ -22,6 +26,9 @@ export class SentryProjectSelector extends BaseStep {
     let selectedProject = null;
     if (answers.wizard.projects.length === 1) {
       selectedProject = { selectedProject: answers.wizard.projects[0] };
+      // the wizard CLI closes too quickly when we skip the prompt
+      // as it will cause the UI to be stuck saying Waiting for wizard to connect
+      await sleep(1000);
```

Comment:
This feels a bit flakey to me. How do we know that 1s is enough? I am not too familiar with the execution flow of the wizard but is there a way to wait more reliably here?

Reply:
@lforst I think the issue is basically waiting for the web page to load. As far as I know, there's no way we can know when that actually opens.

Reply:
The wizard could host a webserver locally that we try to ping from the website via localhost. Just an idea tho. Obviously there would need to be some kind fallback if that's not possible. For example if the wizard runs on some kind of remote machine.

Reply:
@lforst Yea it's an idea. If this simple sleep doesn't always work we can evaluate more complicated and more robust solutions.